desc/compute/utils.py

```diff
@@ -95,6 +95,9 @@ def compute(parameterization, names, params, transforms, profiles, data=None, **
     return data
 
 
+# TODO: can we do the basis change here possibly? instead of repeating it everywhere?
```

Comment:
ah i see you had the same thought.

A few options:
- basically everything is natively in rpz, so we could probably just enforce that all the compute functions return things in rpz and then if need be convert them to xyz here, for the few things natively in xyz it means transforming and transforming back but its just a 3x3 matmul so basically free
- could possibly get rid of the basis option entirely? and just make everything return in rpz and leave it up to the caller to then do any basis changes on top of that for their particular application. this might be the easiest option

Reply:
I was thinking more along the lines of the first option as I do like the convenience of being able to request in either XYZ or RPZ basis. I did not include it here because this PR is really just trying to fix a bug and I can open another PR to then do this change for the basis stuff.

Reply:
I will leave it to a future PR